src/content/docs/infrastructure/host-integrations/host-integrations-list/unix-monitoring-integration.mdx

```diff
@@ -14,8 +14,155 @@ freshnessValidatedDate: never
 
-To get started, see the [Unix Monitoring On-Host Integration README](https://github.com/newrelic/newrelic-unix-monitor) on GitHub.
-
 <Callout variant="tip">
   This integration is released as [open source on Github](https://github.com/newrelic/newrelic-unix-monitor). A change log is also available there for the latest updates. Support for this integration is available with an [Expert Services subscription](https://www.newrelic.com/expertservices).
 </Callout>
+
+## Install the integration 
+
+Follow the steps below to install the integration:
+
+<Steps>
+    <Step>
+## Check compatability [#comp]
+
+* A New Relic account.
+* A Unix server that you want to monitor
+* Java JRE/JDK v1.8 or later
+* Network access to New Relic (proxies are supported, see details below)
+
+You also need a supported operating system:
+
+* AIX 7.0 and above
+* HP-UX 11.0 and above
+* Most flavors of Linux 
+* OSX / MacOS 10.9 ('Mavericks') and above
+* Solaris/SunOS 10.0 and above
+
+    </Step>
+    <Step>
+## Download and prepare the integration [#download]
+
+1. Download the latest version of the Unix Monitor [from the GitHub repo](https://github.com/newrelic/newrelic-unix-monitor/releases/latest).
+2. Copy the integration to the Unix server that you want to monitor and then untar it:
+
+    ```shell
+    tar -xvzf <filename.tar.gz>
+    ```
+
+    </Step>
+    <Step>
+## Configure the integration [#configure-step]
+
+1. Open the config file `config/plugin.json`.
+2. Set `account_id` and `insights_insert_key` to your values.
+
+Here's an example of a complex configuration file. Yours will likely not use every config option available:
+
+    ```yml
+    {
+        "global": {
+            "OS": "auto",
+            "account_id": "insert_your_New_Relic_account_ID_here",
+                    "fedramp": "false",
+            "insights_mode": {
+                "insights_insert_key": "enter_insights_insert_key",
+                "insights_data_center": "US"
+            },
+            "proxy": {
+                "proxy_host": "enter_proxy_host",
+                "proxy_port": 5443,
+                "proxy_username": "enter_proxy_username",
+                "proxy_password": "enter_proxy_password"
+            }
+        },
+        "agents": [
+            {
+                "name": "auto",
+                "static": {
+                    "attribute1": "attribute1_value",
+                    "attribute2": 12345
+                }
+            }
+        ]
+    }
+    ```
+
+You can find more information about all available config options at the bottom of this doc.
+    </Step>
+    <Step>
+## Start up the integartion [#start-integration]
+
+1. Run `./pluginctl.sh start` from your terminal
+2. Check logs in the `logs` directory (or another directory you've configured)for errors. 
+3. Data should now flow into your New Relic account.
```

Comment:
How do I check to see this is happening/working before I go to the next section?